product-development/README.md

```diff
@@ -105,33 +105,19 @@ When a section is done, replace :white_small_square: with :white_check_mark:
 
 ## How an Epic Starts
 
-### DRIs
+### A Team is Assigned to an Epic
 
-Epics are assigned to a
-[DRI](https://medium.com/@mmamet/directly-responsible-individuals-f5009f465da4).
+When assigning a team of Tilters to an epic, we're operating on a few hypotheses:
 
-The DRI is responsible for:
-- maintaining the epic state, description, and attached stories updated as the single source of truth of overall work status
-- reducing scope if additional work is discovered
-- driving the epic to completion
-
-The acceptance criteria sections are _especially critical_, including presenting
-the problem and final implementation in the [weekly epics
-meeting](#weekly-epics-meeting).
-
-### How DRIs are Assigned
-
-When assigning DRIs, we're operating on a few hypotheses:
-
-- Projects are far more likely to succeed when the person working on it feels
+- Projects are far more likely to succeed when the people working on it feel
   connected to the project, and feels it lets them use their skills well.
 
 - Conversely, Tilters should be empowered to try epics they feel strongly about,
```

Comment:
This isn't really something we're doing anymore, given the new system, right? AIUI you can't just pick up an epic anymore, but rather should wait to be assigned by Nick. (Which is fine to try but makes this bullet irrelevant)

Reply:
@maiamcc I think you might be misreading this. 

"empowered" doesn't mean "you can pick up an epic whenever you want". 

Here's the PR where the "empowered" line was added:
https://github.com/tilt-dev/company/commit/c89309e4410a99c56f459f6ec0f58c04725e1ded
It has always been here to say "you're expected to talk to Nick about it if you feel strongly about it".

The empowerment bit was more about "here's a process for how to get this prioritized", rather than people trying to prioritize by consensus, AFAIR